.gitignore

```diff
@@ -3,3 +3,5 @@
 *~
 *.Rproj.user
 *.Rproj
+cso/.DS_Store
+.DS_Store
```

Comment:
We definitely do not want `.DS_Store`, but only the last of these lines should be needed :)